src/data/calculatorPages.js

```diff
@@ -44,21 +44,33 @@ const BODY_TYPES = {
 };
 
 export default {
-    "landing-0": {
-        header: "Misdemeanor Calculator",
-        body: [
-            {
-                type: BODY_TYPES.PARAGRAPH,
-                text: "You can use our eligibility calculator to determine whether you are eligible to vacate your misdemeanor conviction.",
-            }
-        ],
-        buttons: [
-            {
-                text: "Start",
-                href: CALC_PREFIX + "/landing-1"
-            }
-        ],
-        tooltip: "My conviction is not a misdemeanor or I’m not sure.",
+  "landing-0": {
+    header: "Misdemeanor Calculator",
+    body: [
+      {
+        type: BODY_TYPES.PARAGRAPH,
+        text: "You can use our eligibility calculator to determine whether you are eligible to vacate your misdemeanor conviction.",
+      },
+    ],
+    buttons: [
+      {
+        text: "Start",
+        href: CALC_PREFIX + "/landing-1",
+      },
+    ],
+    tooltip: "My conviction is not a misdemeanor or I’m not sure.",
+  },
+  "landing-ineligible-0": {
+    header: "Our calculator can’t determine your eligibility",
+    body: [
+      {
+        type: BODY_TYPES.PARAGRAPH,
+        text: "It looks like your conviction may not be a misdemeanor. Our calculator is unable to assist you with your conviction at this point. We would like to hear your input and learn about your experience as we continue building our calculator.",
+      },
+    ],
+    footerLink: {
+      text: "Take Survey",
+      href: "https://docs.google.com/forms/d/e/1FAIpQLSeuT2oIBnxFAMv6p8tsujNlSugWxFS5la4DUC4rTFekInrcqQ/viewform?usp=sf_link",
     },
```

Comment:
```suggestion
```

This is duplicate code. Might have been added accidentally during the merge conflict resolution.